src/__test__/redux/actions/samples/updateSample.test.js

```diff
@@ -57,4 +73,48 @@ describe('updateSample action', () => {
 
     expect(saveSamples).toHaveBeenCalled();
   });
+
+  it('Dispatches events correctly for api v2', async () => {
+    config.currentApiVersion = api.V2;
+    fetchMock.mockResponseOnce(() => Promise.resolve(JSON.stringify({})));
+
+    const sampleDiff = {
+      name: 'updated name',
+    };
+
+    const store = mockStore(mockState);
+    await store.dispatch(updateSample(mockUuid, sampleDiff));
+
+    const actions = store.getActions();
+    expect(_.map(actions, 'type')).toEqual([SAMPLES_SAVING, SAMPLES_SAVED, SAMPLES_UPDATE]);
+    expect(_.map(actions, 'payload')).toMatchSnapshot();
+
+    expect(fetchMock).toHaveBeenCalledWith(
```

Comment:
why use `toMatchSnapshot` in the previous line and here build this complex object instead?

Reply:
from what I understnad `toMatchSnapshot` is good but I try to avoid abusing it so that I can keep tests readable without having to go to the .snap files all the time

Reply:
ok